fhir-server-test/src/test/resources/test.properties

```diff
@@ -60,3 +60,5 @@ test.bulkdata.export.s3.enabled = false
 # Enable/disable the Azure storage provider tests
 test.bulkdata.import.azure.enabled = false
 test.bulkdata.export.azure.enabled = false
+test.audit.kafka.connectionInfo = localhost:29094,localhost:49098
```

Comment:
it probably makes sense to separate these visually from the bulkdata properties
```suggestion

# Audit log testing parameters
test.audit.kafka.connectionInfo = localhost:29094,localhost:49098
```

Reply:
This has been fixed as suggested. Also made changes to fetch the kafka topic name from "fhir-server-webapp/src/test/liberty/config/config/default/fhir-server-config-audit-cicd.json"